cmake/px4_add_library.cmake

```diff
@@ -44,7 +44,11 @@ function(px4_add_library target)
 	target_compile_definitions(${target} PRIVATE MODULE_NAME="${target}")
 
 	# all PX4 libraries have access to parameters and uORB
-	add_dependencies(${target} uorb_headers parameters)
```

Comment:
Let me see if we can get rid of this entirely.

Reply:
Were you able to determine if this can be removed completely?